pkgs/development/python-modules/x-transformers/default.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pytest-runner,
+  setuptools,
+  wheel,
+  einops,
+  torch,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "x-transformers";
+  version = "1.32.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "x_transformers";
+    inherit version;
+    hash = "sha256-NUIlJ+N2/6kz3rI0oc7bfYVWnMkZhvfljQ6zSwBPkWo=";
+  };
+
+  build-system = [ setuptools ];
+  nativeBuildInputs = [
+    wheel
+    pytest-runner
+  ];
+
+  propagatedBuildInputs = [
+    einops
+    torch
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "x_transformers" ];
+
+  meta = {
+    description = "X-Transformers - Pytorch";
+    longDescription = ''
+      A simple but complete full-attention transformer with a set of promising experimental features from various papers
+    '';
+    homepage = "https://pypi.org/project/x-transformers/";
```

```suggestion
    homepage = "https://github.com/lucidrains/x-transformers";
```
That page doesn't provide any information.